core-services/ci-operator-configresolver/ci-operator-configresolver.yaml

```diff
@@ -44,6 +44,10 @@ items:
     to:
       kind: Service
       name: ci-operator-configresolver
+      weight: 100
```

Comment:
Not aligned with (should be 1 by default, but 100 in production)

```bash
oc explain route.spec.to.weight
KIND:     Route
VERSION:  route.openshift.io/v1

FIELD:    weight <integer>

DESCRIPTION:
     weight as an integer between 0 and 256, default 1, that specifies the
     target's relative weight against other target reference objects. 0
     suppresses requests to this backend.
```

Reply:
I found the cause: `oc-explain` is related to the oc-server version.

```bash
oc --context build01 explain route.spec.to.weight
KIND:     Route
VERSION:  route.openshift.io/v1

FIELD:    weight <integer>

DESCRIPTION:
     weight as an integer between 0 and 256, default 100, that specifies the
     target's relative weight against other target reference objects. 0
     suppresses requests to this backend.

 oc explain route.spec.to.weight
KIND:     Route
VERSION:  route.openshift.io/v1

FIELD:    weight <integer>

DESCRIPTION:
     weight as an integer between 0 and 256, default 1, that specifies the
     target's relative weight against other target reference objects. 0
     suppresses requests to this backend.

```

This is fixed 8 months ago. https://github.com/openshift/api/blame/master/route/v1/types.go#L124

But it is still required on build01 (4.2.10)

```
oc --context build01 explain route.spec.to
KIND:     Route
VERSION:  route.openshift.io/v1

RESOURCE: to <Object>

DESCRIPTION:
     to is an object the route should use as the primary backend. Only the
     Service kind is allowed, and it will be defaulted to Service. If the weight
     field (0-256 default 100) is set to zero, no traffic will be sent to this
     backend.

     RouteTargetReference specifies the target that resolve into endpoints. Only
     the 'Service' kind is allowed. Use 'weight' field to emphasize one over
     others.

FIELDS:
   kind	<string> -required-
     The kind of target that the route is referring to. Currently, only
     'Service' is allowed

   name	<string> -required-
     name of the service/target that is being referred to. e.g. name of the
     service

   weight	<integer> -required-
     weight as an integer between 0 and 256, default 100, that specifies the
     target's relative weight against other target reference objects. 0
     suppresses requests to this backend.
```

It became `optional` last month
https://github.com/openshift/api/blame/master/route/v1/types.go#L126